.github/workflows/firebase-hosting-merge.yml

```diff
@@ -30,3 +30,9 @@ jobs:
           projectId: project-hub-116d7
         env:
           FIREBASE_CLI_PREVIEWS: hostingchannels
+      - name: ESLint checker
```

Comment:
It might be cleaner at some point to separate this into its own file and run multiple GitHub actions side by side

Reply:
Yeah totally man, thanks for pitching in here :)